src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -64,14 +65,26 @@ const WorkspacePermissionNotice = ({ workspace }) => {
   ])
 }
 
+const AzureWarning = () => {
+  const warningMessage = 'Reminder: Azure workspaces (like this one) are currently NOT authorized to contain any federally protected data. ' +
+    'Please do not import or use any federally protected data in Azure workspaces.'
+
+  return h(TitleBar, {
+    title: div({ style: { display: 'flex', alignItems: 'center', margin: '1rem' } }, [
```

Comment:
I would recommend adding a role of alert (`div({ role: 'alert'…`), which also addresses this accessibility flag:

![image](https://user-images.githubusercontent.com/484484/205081660-89538b06-f703-4cf0-8445-0d0fb4a89d15.png)

Reply:
Ah good catch -- I didn't see that on Lighthouse. What tool is that?

Reply:
It's axe dev tools. You can choose to install and run it on demand, but it also executes automatically for you when you are running Terra locally-- look in the dev console. There's a possibility that this particular one doesn't show in that environment though, as it's a "best-practice" vs. a WCAG requirement.

Reply:
See https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide#accessibility for axe link. And thanks for running Lighthouse!

Reply:
ah cool, I see that in the console! Thanks Christina!